Documented how normals interact with real bounds. (#19)
CitrineInformatics/taurus-documentation: docs/specification/attribute-templates.md

Real bounds can validate distributions over the real numbers.

> As a general principle, distributions do not need to be zero for all points outside the associated real bounds.

Reviewer: ```suggestion
A distribution may be non-zero for points outside of the associated real bounds and still be valid.
```

> This is because common distributions such as a Gaussian are greater than zero everywhere,

Reviewer: ```suggestion
This is because common distributions such as a Gaussian are non-zero everywhere.
```

> and would therefore always be invalid.

Reviewer: ```suggestion
```
I think the point is clear without this additional clause.
